### DISCUSSION TITLE
Help!I forked and import it but it did not work

### DISCUSSION THREAD

**Discussion by BolshevikNanven** [spencerwooo/onedrive-vercel-index#381] [Q&A] [answered]
Running "yarn run build"
yarn run v1.22.17
$ next build
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry
info  - Checking validity of types...
Failed to compile.
./components/FileListing.tsx:78:22
Type error: Type '[IconPrefix, IconName]' is not assignable to type 'IconProp'.
  76 |     <span>{emoji ? emoji[0] : '📁'}</span>
  77 |   ) : (
> 78 |     <FontAwesomeIcon icon={child.file ? getFileIcon(child.name, { video: Boolean(child.video) }) : ['far', 'folder']} />
     |                      ^
  79 |   )
  80 | }
  81 | 
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1

**Comment by myl7** [comment, marked as answer]
See discussion #374 , you may need to change the build and install command in Vercel console.

**Reply by BolshevikNanven** [reply]
thanks
